docs/Overrides/NginxProxyManager.md

```diff
@@ -0,0 +1,98 @@
+# Nginx Proxy Manager
+[Nginx Proxy Manager](https://[https://nginxproxymanager.com/](https://nginxproxymanager.com/))  is  Docker container for managing Nginx proxy hosts and SSL Certificates with a simple, powerful interface.
+ 
+# Features of Nginx Proxy Manager
+ 
+### Get Connected
+Expose web services on your network · Free SSL with Let's Encrypt · Designed with security in mind · Perfect for home networks
+
+### Proxy Hosts
+Expose your private network Web services and get connected anywhere.
+
+### Beautiful UI
+Based on Tabler, the interface is a pleasure to use. Configuring a server has never been so fun.
+
+### Free SSL
+Built in Let’s Encrypt support allows you to secure your Web services at no cost to you. The certificates even renew themselves!
+
+### Multiple Users
+Configure other users to either view or manage their own hosts. Full access permissions are available.
+
+The GIT Repository for Nginx Proxy Manager is located at [[https://github.com/jc21/nginx-proxy-manager](https://github.com/jc21/nginx-proxy-manager)]
+### Config File
+Nginx Proxy Manager requires a Configuration file named `config.json`. This file needs to be in the appdata folder for NPM before it is started.
+
+```json
+{
+  "database": {
+    "engine": "mysql",
+    "host": "db",
+    "name": "npm",
+    "user": "npm",
+    "password": "npm",
+    "port": "3306"
+  }
+}
+```
+## Example Docker Compose Override
+
+```yaml
+#NGINX Proxy Manager with LetsEncrypt https://github.com/jc21/nginx-proxy-manager
+    version: "3.4"  # this must match the version in docker-compose.yml
+    services:
+        proxymanager:
+        image: jc21/nginx-proxy-manager:latest
+        container_name: proxymanager
+        labels:
+          - "com.dockstarter.appinfo.description: NGINX Proxy Manager with LetsEncrypt included"
+          - "com.dockstarter.appinfo.nicename: NGINX Proxy Manager"
+        logging:
+          driver: json-file
+          options:
+            max-file: ${DOCKERLOGGING_MAXFILE}
+            max-size: ${DOCKERLOGGING_MAXSIZE}
+        ports:
+          - "80:80"
+          - "81:81"
+          - "443:443"
+        environment:
+          - FORCE_COLOR=1
+          - NODE_ENV=config
+        volumes:
+          - /opt/appdata/proxymanager/config.json:/app/config/config.json
+          - /opt/appdata/proxymanager/data:/data
+          - /opt/appdata/proxymanager/letsencrypt:/etc/letsencrypt
+          - ${DOCKERSHAREDDIR}:/shared
+        depends_on:
+          - mariadb
+        restart: unless-stopped
+    
+      mariadb:
```

Comment:
mariadb is available to be run via DS and does not need to be included in the override. Documenting that this app requires a mysql/mariadb database and information about how to enable it in DS and setup a database and user (usually with phpmyadmin) would be a better option. The topic of setting up a database and user is fairly common and maybe should be documented on either the mariadb or phpmyadmin app pages that already exist in the repo, and then pages like this could just link to that information.

Reply:
I can do the documenting for those pages. Gives me a reason to play with those two containers. I’ll push a PR for updating those pages in the next few days.

Reply:
i think maybe an FAQ page on MDb and PMA and how you can use PMA to not have to have multiple containers that is linked on both MDb and PMA pages would be better

Reply:
Let me play with the containers so what you’re saying sort of clicks with me. I know enough in SQL to break stuff but not fix it 🤣

Reply:
you know more than me! lol

Reply:
So we can definitely put a disclaimer on either PMA's or MDb's for managing databases but I think it would be a good idea to include a how to create a DB on each app's page